### PR TITLE
added kwargs for softmax_cross_entropy_with_logits calls

### PR DESCRIPTION
**Pull request by yufengg**

### CODE REVIEW

**Comment by ChickenProp** [comment]
This commit breaks the project for me. It looks like the correct name for the kwarg is, and always has been, 'labels' rather than 'targets'. https://www.tensorflow.org/api_docs/python/nn/classification#softmax_cross_entropy_with_logits

**Comment by martin-gorner** [comment]
indeed, I fixed it in master.
